src/components/Skeleton/Skeleton.jsx

```diff
@@ -59,7 +59,7 @@ const Skeleton = (props) => {
     axios
       .get("/json/logininfo")
       .then(({ data }) => {
-        setUserId(data.id ? data.id : null);
+        setUserId(data.id ?? null);
```

Comment:
```suggestion
        setUserId(data?.id ?? null);
```
로 해야 하지 않을까요? 그리고 응답이 왔는데 이런 경우가 있을까요...?

Reply:
생각해보니 어차피 data가 `null`이면 `null`을 반환하니까 굳이 `??`를 붙힐 필요가 없네요.

Reply:
아 아니네요 data가 null이면 data?.id가 undefined를 반환하는데 저희가 지금 로그인이 안된 상태에서는 userId가 `null`이어야 하고 로그인 직후 로딩 중애는 `undefined`인 상태라 `null`을 반환하도록 처리해줘될 것 같아요. 저렇게 처리 안해둔 상태로 data?.id로만 두면 혹시 백에서 오류가 나는 경우에 화면에 아무것도 안 뜨게 될 것 같아요